bot/exts/moderation/infraction/_utils.py

```diff
@@ -27,16 +27,18 @@
 # Type aliases
 Infraction = t.Dict[str, t.Union[str, int, bool]]
 
-APPEAL_EMAIL = "appeals@pythondiscord.com"
+APPEAL_SERVER_INVITE = "https://discord.gg/WXrCJxWBnm"
 
 INFRACTION_TITLE = "Please review our rules"
-INFRACTION_APPEAL_EMAIL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_APPEAL_SERVER_FOOTER = f"\n\nTo appeal this infraction, join our [appeals server]({APPEAL_SERVER_INVITE})."
 INFRACTION_APPEAL_MODMAIL_FOOTER = (
-    'If you would like to discuss or appeal this infraction, '
-    'send a message to the ModMail bot'
+    '\n\nIf you would like to discuss or appeal this infraction, '
+    'send a message to the ModMail bot.'
```

Comment:
Wouldn't this be the "Ban Appeals" bot rather than ModMail? Potentially also means constant should be renamed to `INFRACTION_BAN_APPEAL_BOT_FOOTER` or alike

Reply:
`INFRACTION_APPEAL_SERVER_FOOTER` is used for bans, `INFRACTION_APPEAL_MODMAIL_FOOTER` is used for everything else.

The reason for this is because banned used cannot DM modmail, as they do not share a server, so they need to join the appeal server to be able to. Whereas other kinds of infractions don't get kicked from the server, so they have access to DM modmail directly.

Reply:
Do we currently handle the situation for kicks? Whilst it's rare that we kick instead of mute/ban someone, it does sometimes happen and so this should be handled. Presumably we'd want a different message saying "rejoin when you're ready to follow our rules and coc" or something.

Reply:
Kicks currently also send the `INFRACTION_APPEAL_MODMAIL_FOOTER` we could have another message that explains it more along the lines that you suggest. But that's out of scope for this PR, so should be another issue/PR.